flopy/utils/gridintersect.py

```diff
@@ -557,7 +599,12 @@ def _intersect_point_shapely(
                     else cell_shps[0]
                 )
                 vertices.append(tuple(cell_verts))
-                cellids.append(name)
+                # if structured calculated (i, j) cell address
+                if self.mfgrid.grid_type == "structured":
+                    i = cid // self.mfgrid.ncol
+                    j = cid - i * self.mfgrid.ncol
+                    cid = (i, j)
```

Comment:
Recommend changing this to:
```python
cid = self.mfgrid.get_lrc([cid])[1:]
```
to avoid code duplication.

Reply:
Thanks! Replaced the code with your suggestion.